authzed/api/v1/permission_service.proto

```diff
@@ -362,6 +369,9 @@ message LookupSubjectsRequest {
     pattern : "^([a-z][a-z0-9_]{1,62}[a-z0-9])?$",
     max_bytes : 64,
   } ];
+
+  /** context consists of named values that are injected into the caveat evaluation context **/
+  google.protobuf.Struct context = 6 [ (validate.rules).message.required = false ];
 }
 
 // LookupSubjectsResponse contains a single matching subject object ID for the
```

Comment:
We need to add an enum here to indicate that the response is either `HAS_PERMISSION` or `MAYBE_PERMISSION`, along with the same "missing fields"